code/obj/machinery/computer/announcement.dm

```diff
@@ -213,3 +214,31 @@
 		name = "Syndicate Announcement computer"
 		voice_name = "Syndicate Announcement Computer"
 		theme = "syndicate"
+
+/obj/machinery/computer/announcement/clown
+	req_access = null
```

Comment:
There's no such thing as clown access buuut maybe you could make it require a clown ID since that's unique by itself? I'm just thinking about random staffies wandering in and using the one announcement instead.

Reply:
alternatively it could require clown training (and maybe mime training could announce "...")

Reply:
I'd prefer tying it to something tangible rather than hidden training properties. If you wanted to be really mean you could make it shred the ID too, since it's canonically made of cardboard. (maybe an emag effect?)